saleor/order/calculations.py

```diff
@@ -0,0 +1,261 @@
+from decimal import Decimal
+from typing import Iterable, Optional, Tuple
+
+from django.conf import settings
+from django.db.models import prefetch_related_objects
+from django.utils import timezone
+from prices import Money, TaxedMoney
+
+from ..core.prices import quantize_price
+from ..core.taxes import TaxData, TaxError, TaxLineData, zero_taxed_money
+from ..plugins.manager import PluginsManager
+from . import ORDER_EDITABLE_STATUS
+from .models import Order, OrderLine
+
+
+def _combine_prices_from_manager_into_tax_data(
+    manager: PluginsManager, order: Order, lines: Iterable[OrderLine]
+) -> Optional[TaxData]:
+    prefetch_related_objects(lines, "variant__product")
+    currency = order.currency
+
+    tax_lines_data = []
+    subtotal = zero_taxed_money(currency)
+    try:
+        for line in lines:
+            variant = line.variant
+            if not variant:
```

Comment:
I think that you should include the line prices to subtotal. Even if you're not able to update prices for given line.